plugins/kubernetes-provenance/Makefile

```
@@ -0,0 +1,18 @@
+
+.PHONY: build
+build:
+	@test -n "$(PLUGIN_PATH)" || (echo "MISSION_CONTROL_PLUGIN_PATH is required"; exit 1)
+	mkdir -p "$(PLUGIN_PATH)"
+	go build \
+		-ldflags "-X 'main.Version=$(VERSION)' -X 'main.BuildDate=$(BUILD_DATE)'" \
+		-o "$(PLUGIN_PATH)/$(PLUGIN_NAME)" \
+		"$(PLUGIN_DIR)"
+
+.PHONY: install
+install: build
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)